CHANGELOG.md

```diff
@@ -2,10 +2,31 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.5 July 28, 2021
+# 10.0.0-rc.6 August 11, 2021
 
 **The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
 
+## Breaking changes ⚠️
+* Update extension function signatures making them easier to use from Java. ([#539](https://github.com/mapbox/mapbox-maps-android/pull/539))
+* Rename `mapView#overlay()` to `mapView#mapboxOverlay`. ([#539](https://github.com/mapbox/mapbox-maps-android/pull/539))
+
+## Features ✨ and improvements 🏁
+* Support adding 9-patch images to the style. ([#536](https://github.com/mapbox/mapbox-maps-android/pull/536))
+* Volatile geometry tiles do not show outdated data, they get hidden if the network refresh fails. ([#543](https://github.com/mapbox/mapbox-maps-android/pull/543))
```

Comment:
this entry doesn't sound user-facing, shall we ask the native team to provide more customer facing change log entries?

Reply:
How can we make it more user-facing in your opinion?
Seems pretty clear to me.